lib/lotus/routing/force_ssl.rb

```diff
@@ -0,0 +1,93 @@
+module Lotus
+  module Routing
+    # Force ssl
+    #
+    # Redirect response to the secure equivalent resource (https)
+    #
+    # @since x.x.x
+    # @api private
+    class ForceSsl
+      # Https scheme string
+      #
+      # @since x.x.x
+      # @api private
+      SSL_SCHEME = 'https'.freeze
+
+      # Location header string
+      #
+      # @since x.x.x
+      # @api private
+      LOCATION_HEADER = 'Location'.freeze
+
+      # Default port number
+      #
+      # @since x.x.x
+      # @api private
+      DEFAULT_PORT = 80
```

Comment:
@AlfonsoUceda When you use SSL via browser, it uses 443, even if you don't see it.